mail-filter/rspamd/rspamd-3.0-r3.ebuild

```diff
@@ -35,6 +35,7 @@ RDEPEND="${LUA_DEPS}
 	acct-user/rspamd
 	app-arch/zstd:=
 	dev-db/sqlite:3
+	dev-libs/doctest:=
```

Comment:
You're not exposing a subslot in doctest?

Reply:
Yeah, the thing with this is that I wasn't sure how I should expose the subslot (how many version numbers), but I forgot to mention it in PR. It is a single-header C++ library with a lot of templates, therefore, I decided `0/${PV}` should be there, similar to boost.

Reply:
If it's header-only, we should be fine _not_ to (the problem with Boost is that Boost has some parts which must be linked against, and figuring out if you do/don't need them is tricky).

I'll drop it for now, I think.

Reply:
Actually, this makes me think about putting it in dev-cpp.

Reply:
Ok, thanks! Anyway, my idea behind slotting was that in case of some bug it is necessary to rebuild packages depending on `doctest`. But we can introduce it if it really will be needed in future.